utils/rpms/daos.spec

```diff
@@ -164,6 +165,7 @@ Requires: ipmctl > 02.00.00.3816
 Requires: libpmem >= 1.8, libpmemobj >= 1.8
 %endif
 Requires: hwloc
+Requires: tbb
```

Comment:
This shouldn't be necessary.  RPM will detect that libtbb.so.* was linked to and automatically create an implicit `Requires: tbb`.  Relying on this rather than explicitly adding it here allows the libtbb maintainer to make changes to the packaging of the tbb RPM without breaking our packages.

Reply:
removed